app/(header-default)/entreprise/[slug]/page.tsx

```diff
@@ -66,12 +66,21 @@ export default async function UniteLegalePage(props: AppRouterProps) {
   ]);
 
   const extractedSiren = extractSirenOrSiretSlugFromUrl(slug);
+  // We redirect from /entreprise/${siren} to /entreprise/${slug}
+  // Nb: in somes cases, there can be a two redirects :
+  // /rechercher?terme=${siren} -> /entreprise/${siren}?isRedirected=1 -> /entreprise/${slug}?isRedirected=1
```

Comment:
Indeed this is better like this !

Just a little change :

```suggestion
  // /rechercher?terme=${siren} -> /entreprise/${siren}?redirected=1 -> /entreprise/${slug}?redirected=1
```